.changeset/cuddly-clouds-complain.md

```diff
@@ -0,0 +1,7 @@
+---
+'@astrojs/cloudflare': major
```

Comment:
If this only fixes bugs then this shouldn't be a major. Maybe you were just being conservation here, or can you tell me what this breaks?

If this is a true breaking change I think we should have more explanation of the change in the changeset.

Reply:
adjusted this one to minor